src/rules/at-import-no-partial-leading-underscore/index.js

```diff
@@ -16,6 +16,16 @@ const meta = {
 
 function rule(actual) {
   return (root, result) => {
+    result.warn(
+      "'at-import-no-partial-leading-underscore' has been deprecated, " +
+        "and will be removed in '6.0'. Use 'load-no-partial-leading-underscore' instead.",
+      {
+        stylelintType: "deprecation",
+        stylelintReference:
+          "https://github.com/stylelint-scss/stylelint-scss/tree/master/src/rules/load-no-partial-leading-underscore/README.md"
```

Comment:
This deprecation message looks nice, but just a couple of small things to fix.

From the developer guide:

> Deprecating rules doesn't happen very often. When you do, you must:
>
> 1. Point the `stylelintReference` link to the specific version of the rule README on the GitHub website, so that it is always accessible.
> 2. Add the appropriate metadata to mark the rule as deprecated like `rule.meta = { deprecated: true }`.

`stylelintReference`  should probably point to the old rule, not the new one, and the url should be locked to a version (e.g. current release). You can use this for reference:
https://github.com/stylelint/stylelint/blob/13.7.0/lib/rules/at-rule-blacklist/index.js#L32C1-L35C6

Also,`deprecated: true` needs to be added to the `meta` object in the rule.

Reply:
Gotcha, sorry I missed that! Thank you